assets/base/i18n/en.json

```diff
@@ -374,6 +374,8 @@
   "mobile.calls_lasted": "Lasted {duration}",
   "mobile.calls_leave": "Leave",
   "mobile.calls_leave_call": "Leave call",
+  "mobile.calls_limit_msg": "Maximum number of participants per call is currently limited to {maxParticipants}. Please contact your system admin to increase the limit.",
```

Comment:
```suggestion
  "mobile.calls_limit_msg": "The maximum number of participants per call is {maxParticipants}. Contact your System Admin to increase the limit.",
```

Reply:
Is there a CTA for "Contact your System Admin"?

Reply:
Not in the alert itself, we can add that if we need to in another PR (but I think it might be too much -- there's no flow for that in mobile yet).